python/taichi/core/profiler_modes.py

```diff
@@ -0,0 +1,20 @@
+from taichi.core.util import ti_core as _ti_core
+
+# Kernel Profiler Mode
+kernel_profiler_disable = _ti_core.KernelProfilerMode.disable
+kernel_profiler_enable = _ti_core.KernelProfilerMode.enable
+cuda_accurate = _ti_core.KernelProfilerMode.cuda_accurate
+cuda_detailed = _ti_core.KernelProfilerMode.cuda_detailed
+
+kernel_profiler_enums = [
+    kernel_profiler_disable, kernel_profiler_enable, cuda_accurate,
+    cuda_detailed
+]
+kernel_profiler_enums_ids = [id(t) for t in kernel_profiler_enums]
+
+__all__ = [
+    'kernel_profiler_disable',
```

Comment:
Hmmm personally I'd prefer these enum names to be more explicit, e.g. `DISABLED, CUEVENT, CUPTI_ONEPASS, CUPTI_DETAILED` so that I know what I'm calling (and potentially easier to add new profiling modes in the future ;) Wdyt?

Reply:
Of course. Enum names have not yet been finalized, perhaps a short discussion is needed.

Reply:
`disable` and `enable` are used for compatibility with the current `True` and `False`. In general, we can follow the current method: `kernel_profiler = True`

Reply:
For CUDA backend, `enable` and `CUEVENT` (default kernel profiler) are equivalent, we can also add this enum name to explicitly specify CUEVENT profiler.